valid_angle_range_list: [rad(-10.0*pi/16.0), rad(-5.0*pi/16.0),
                         rad(-1.4*pi/16.0), rad(1.5*pi/16.0),
                         rad(5.0*pi/16.0), rad(10.0*pi/16.0)]
